pkg/appmgmt/general/general.go

```diff
@@ -113,7 +113,7 @@ func (os *Manager) getTaskMetadata(pod *v1.Pod) (interfaces.TaskMetadata, bool)
 	}, true
 }
 
-func (os *Manager) getAppMetadata(pod *v1.Pod, recovery bool) (interfaces.ApplicationMetadata, bool) {
+func (os *Manager) GetAppMetadata(pod *v1.Pod, recovery bool) (interfaces.ApplicationMetadata, bool) {
```

Comment:
Changed the visibility to be consistent with `GetTaskMetaData()`, but not strictly necessary.